src/config/app.config.ts

```diff
@@ -1,46 +1,73 @@
 import { registerAs } from '@nestjs/config';
+import { APPLICATION } from '@ukef/constants';
 import { getIntConfig } from '@ukef/helpers/get-int-config';
 
 import { InvalidConfigException } from './invalid-config.exception';
 
+const { NODE_ENV } = process.env;
+
+const { NAME, VERSION_PREFIX } = APPLICATION;
+
 const validLogLevels = ['fatal', 'error', 'warn', 'info', 'debug', 'trace', 'silent'];
 
 export interface AppConfig {
-  name: string;
+  apiKey: string;
   env: string;
-  versioning: {
+  name: string;
+  globalPrefix: string;
+  logLevel: string;
+  odsVersioning: {
     enable: boolean;
     prefix: string;
     version: string;
+    prefixAndVersion: string;
   };
-  globalPrefix: string;
   port: number;
-  apiKey: string;
-  logLevel: string;
   redactLogs: boolean;
   singleLineLogFormat: boolean;
+  versioning: {
+    enable: boolean;
+    prefix: string;
+    version: string;
+  };
 }
 
 export default registerAs('app', (): Record<string, any> => {
   const logLevel = process.env.LOG_LEVEL || 'info';
+
   if (!validLogLevels.includes(logLevel)) {
     throw new InvalidConfigException(`LOG_LEVEL must be one of ${validLogLevels} or not specified.`);
   }
-  return {
-    name: process.env.APP_NAME || 'mdm',
-    env: process.env.APP_ENV || 'development',
 
-    versioning: {
-      enable: process.env.HTTP_VERSIONING_ENABLE === 'true',
-      prefix: 'v',
-      version: process.env.HTTP_VERSION || '1',
-    },
+  const mdmVersion = process.env.HTTP_VERSION || '1';
 
-    globalPrefix: '/api',
-    port: getIntConfig(process.env.HTTP_PORT, 3003),
+  const mdmVersioning = {
+    enable: process.env.HTTP_VERSIONING_ENABLE === 'true',
+    prefix: VERSION_PREFIX,
+    prefixAndVersion: `${VERSION_PREFIX}${mdmVersion}`,
+    version: mdmVersion,
+  };
+
+  const odsVersion = process.env.ODS_HTTP_VERSION;
+
+  const odsVersioning = {
+    enable: process.env.HTTP_VERSIONING_ENABLE === 'true',
+    prefix: VERSION_PREFIX,
+    prefixAndVersion: `${VERSION_PREFIX}${odsVersion}`,
+    version: odsVersion,
+  };
+
+  return {
     apiKey: process.env.API_KEY,
+    env: NODE_ENV || 'development',
```

Comment:
```suggestion
    env: NODE_ENV || ENV.DEV,
```

Can this please be a constant, use `DTFS` or `EXIP` constant sytle please.